src/linter/linter.ts

```diff
@@ -114,11 +118,12 @@ export async function lintFile({
 }
 
 async function lint(
-	resourceReader: AbstractReader, options: LinterOptions, config: UI5LintConfigType
+	resourceReader: AbstractReader, options: LinterOptions & {relFsBasePath: string; virBasePath: string},
```

Comment:
Why is the `& {relFsBasePath: string; virBasePath: string}` needed? Is it because those are mandatory here, but optional in the `LinterOptions`?

Reply:
Now it's refactored